lib/lightning/jobs/query.ex

```diff
@@ -34,11 +34,9 @@ defmodule Lightning.Jobs.Query do
   @spec enabled_cron_jobs_by_edge() :: Ecto.Queryable.t()
   def enabled_cron_jobs_by_edge do
     from(e in Edge,
-      join: j in Job,
-      on: e.target_job_id == j.id,
-      join: t in Trigger,
-      on: t.id == e.source_trigger_id,
-      where: t.type == :cron,
+      join: j in assoc(e, :target_job),
+      join: t in assoc(e, :source_trigger),
+      where: t.type == :cron and j.enabled,
```

Comment:
@stuartc  Since this returns an edge, Don't you think moving it to worflows is better? keeping it in jobs is confusing?

Reply:
Yeah that makes sense to me.